### DISCUSSION TITLE
Disable nightly

### DISCUSSION THREAD

**Discussion by cosmincatalin** [topgrade-rs/topgrade#924] [Q&A] [answered]
Hi,

Is there a way to disable updating `rust` nightly?

**Comment by SteveLauC** [comment, marked as answer]
Hi, AFAIK, nope, but I think we should implement this feature 🤔

**Comment by SteveLauC** [comment]
Opened a feature request issue for this: https://github.com/topgrade-rs/topgrade/issues/936
